test/libsolidity/semanticTests/modifiers/break_in_modifier.sol

```diff
@@ -3,7 +3,8 @@ contract C {
     modifier run() {
         for (uint256 i = 0; i < 10; i++) {
             _;
-            break;
+            if (i == 1)
```

Comment:
Without this change, this test passed, because simply the modifier statements were not compiled.